utils/default-helpers/src/test/scala/pl/touk/nussknacker/engine/util/functions/NumericUtilsSpec.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.engine.util.functions
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import cats.implicits._
+
+import java.math.BigInteger
+
+class NumericUtilsSpec extends AnyFunSuite with BaseSpelSpec with Matchers {
+
+  test("numeric") {
```

Comment:
I would call it `test("string to number conversion should return narrow numeric types in the runtime")` or sth similar. Thanks to that we would know in the future, why we introduced this change.